File: web-development/level-1/helsinki-part-1-2/Python/part1/week02/ex34.py
#ex 34
#Factorial

print('Facotring')

inputNum = int(input('Type a number: '))
sumNum = 1

if inputNum == 0:
    print('Sum is : 1')

i = 1
while i <= inputNum:
    sumNum *= i
    if i >= inputNum:
        print('Sum is : ', sumNum )
        break
    i += 1

File: programming/helsinki-moocs/java/ex34.java
import java.util.Scanner;

public class ex34 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    System.out.print("Factorial n: ");
    int n = Integer.parseInt(input.nextLine());

    int sum = 1;
    int i = 1;
    while(i <= n) {
      sum *= i;
      i++;
    }

    System.out.println("Factorial: "+ sum);


  }// entry point
}
